garage/src/pages/RigDetails/index.tsx

```diff
@@ -287,10 +285,10 @@ export const RigDetails = () => {
                   <RigHeader
                     {...MODULE_PROPS}
                     rig={rig}
-                    owner={owner}
-                    tokenURI={tokenURI}
+                    owner={(owner as any)?.result}
```

Comment:
And then this was in attempt to resolve `Type '{ error: Error; result?: undefined; status: "failure"; }' is not assignable to type 'string'` but looks wrong